waspc/src/Wasp/Generator/ServerGenerator/OperationsRoutesG.hs

```diff
@@ -65,9 +65,10 @@ genOperationRoute spec operation tmplFile = return $ C.mkTmplFdWithDstAndData tm
           baseTmplData
 
     operationImportPath =
-      SP.fromRelFileP $
-        relPosixPathFromOperationsRoutesDirToSrcDir
-          </> fromJust (SP.relFileToPosix $ operationFileInSrcDir operation)
+      C.toESModulesImportPath $
```

Comment:
Why do we need this here?

Reply:
Talked about it, I'll document it above the function.